django/db/models/sql/constants.py

```diff
@@ -35,3 +35,8 @@
 # SQL join types.
 INNER = 'INNER JOIN'
 LOUTER = 'LEFT OUTER JOIN'
+
+# Set operators
+UNION = 'union'
+INTERSECT = 'intersect'
+EXCEPT = 'except'
```

Comment:
I think that we should move set operators to `DatabaseWrapper` for each backend, just like standard operators, e.g. for Oracle backend:
```diff
diff --git a/django/db/backends/oracle/base.py b/django/db/backends/oracle/base.py
index 2e6f02c..ad354c4 100644
--- a/django/db/backends/oracle/base.py
+++ b/django/db/backends/oracle/base.py
@@ -140,6 +140,12 @@ class DatabaseWrapper(BaseDatabaseWrapper):
         'iendswith': "LIKE UPPER(TRANSLATE(%s USING NCHAR_CS)) ESCAPE TRANSLATE('\\' USING NCHAR_CS)",
     }
 
+    _set_operators = {
+        'union': 'UNION ALL',
+        'intersect': 'INTERSECT',
+        'except': 'MINUS',
+    }
+
     _likec_operators = _standard_operators.copy()
     _likec_operators.update({
         'contains': "LIKEC %s ESCAPE '\\'",
```